rules.d/60-autosuspend.rules

```diff
@@ -6,6 +6,18 @@ ACTION!="add", GOTO="autosuspend_end"
 SUBSYSTEM=="i2c", ATTR{name}=="cyapa", \
   ATTR{power/control}="on", GOTO="autosuspend_end"
 
+# Enable autosuspend for qemu emulated usb hid devices
+#
+# Note that there are buggy qemu versions (0.13 & older) which
+# advertise remote wakeup support but don't actually implement
+# it correctly.  This is the reason why we need a match for the
+# serial number here.  Old, broken versions have serial "1".
+# It has been changed to "42" after fixing the bug to indicate
+# remote wakeup is working.
+SUBSYSTEM=="usb", ATTR{product}=="QEMU USB Mouse", ATTR{serial}!="1", TEST=="power/control", ATTR{power/control}="auto"
+SUBSYSTEM=="usb", ATTR{product}=="QEMU USB Tablet", ATTR{serial}!="1", TEST=="power/control", ATTR{power/control}="auto"
+SUBSYSTEM=="usb", ATTR{product}=="QEMU USB Keyboard", ATTR{serial}!="1", TEST=="power/control", ATTR{power/control}="auto"
```

Comment:
I think it would be better to se ID_AUTOSUSPEND here, and let the rule below then set the attribute